src/pybind/mgr/cephadm/inventory.py

```diff
@@ -66,6 +67,115 @@ def __init__(
         self.hostname = hostname
 
 
+class DaemonDeployQueue:
+    """
+    Queue of daemons cephadm thinks it should deploy
+    Note this only covers fresh deployments, not redeploy/reconfig/restart
```

Comment:
can we add to this comment how other cases (redeploy/reconfig) are expected to be deployed?